samples/PowerUtils.AspNetCore.ErrorHandler.Samples/Startup.cs

```diff
@@ -4,79 +4,89 @@
+                .AddAuthentication(option =>
+                {
+                    option.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                    option.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+
+                }).AddJwtBearer(options
+                    => options.TokenValidationParameters = new TokenValidation()
+                );
+
+            // Validation with basic authentication
+            services
+                .AddAuthentication(option =>
+                {
+                    option.DefaultAuthenticateScheme = BasicAuthentication.AUTHENTICATION_SCHEME;
+                    option.DefaultChallengeScheme = BasicAuthentication.AUTHENTICATION_SCHEME;
+                })
+                .AddScheme<AuthenticationSchemeOptions, BasicAuthenticationHandler>(BasicAuthentication.AUTHENTICATION_SCHEME, null);
+
+            services.AddErrorHandler(); // To test
+            services.AddErrorHandler(options =>
             {
-                option.DefaultAuthenticateScheme = BasicAuthentication.AUTHENTICATION_SCHEME;
-                option.DefaultChallengeScheme = BasicAuthentication.AUTHENTICATION_SCHEME;
-            })
-            .AddScheme<AuthenticationSchemeOptions, BasicAuthenticationHandler>(BasicAuthentication.AUTHENTICATION_SCHEME, null);
+                options.PropertyNamingPolicy = PropertyNamingPolicy.SnakeCase;
 
+                options.ExceptionMapper<ModelStatesException>(exception => 599); // Only to test the override a mapping
+                options.ExceptionMapper<ModelStatesException>(exception => (exception.Status, exception.Errors));
 
-        services.AddErrorHandler(); // To test
-        services.AddErrorHandler(options =>
-        {
-            options.PropertyNamingPolicy = PropertyNamingPolicy.SnakeCase;
+                options.ExceptionMapper<CustomException>(exception => 582);
 
-            options.ExceptionMapper<ModelStatesException>(exception => 599); // Only to test the override a mapping
-            options.ExceptionMapper<ModelStatesException>(exception => (exception.Status, exception.Errors));
+                options.ExceptionMapper<TestException>(exception => StatusCodes.Status503ServiceUnavailable);
+                options.ExceptionMapper<TimeoutException>(exception => StatusCodes.Status504GatewayTimeout);
+            });
 
-            options.ExceptionMapper<TestException>(exception => StatusCodes.Status503ServiceUnavailable);
-            options.ExceptionMapper<TimeoutException>(exception => StatusCodes.Status504GatewayTimeout);
-        });
 
+            services.Configure<ApiBehaviorOptions>(options
+                => options.ClientErrorMapping.Add(582, new()
+                {
+                    Link = "CustomLink",
+                    Title = "CustomTitle"
+                })
+            );
 
+            // Configurations to payload size
+            services.Configure<FormOptions>(options =>
```

Comment:
Interesting configuration, i rather like this!

Reply:
We need to find a better way to catch errors when the payload exceeds the allowed limit.